src/main/windows/setting.js

```diff
@@ -30,9 +30,7 @@ class SettingWindow extends BaseWindow {
 
     // WORKAROUND: Electron has issues with different DPI per monitor when
     // setting a fixed window size.
-    if (isWindows) {
```

Comment:
Yes, the window was resizeable on Windows due to another issue in the past. The linked issue is about the missing scrollbar, but I guess no one knows that the windows is resizeable because the window is frameless and hasn't a native style.